app/views/stories/_form_errors.html.erb

```diff
@@ -1,24 +1,25 @@
 <div class="form_errors_header">
-  <% if story.errors.count == 1 && story.already_posted_story %>
+  <% if story.errors.count == 1 && !story.already_posted_stories.empty? %>
     <div class="flash-error">
-    <h2>Error: This story was already submitted <%=
-      time_ago_in_words_label(story.already_posted_story.created_at) %></h2>
+    <h2>Error: This story was most recently submitted <%=
+      time_ago_in_words_label(story.already_posted_stories.first.created_at) %></h2>
     <p>
-    Please view the <a href="<%= story.already_posted_story.comments_path %>"
-    target="_blank">previous discussion</a> for this story.
+    Please view the previous discussions for this story: <br>
```

Comment:
Visual styling: please move the help text + partial render down out of the flash. The screenshots (thanks for including those, super helpful!) show it's too darn much red and blue for our very low-intensity site.

Reply:
so do you just want it to be like a the ordered list on the comments page but between the flash and the submit form?

Reply:
Yep!